File: IfElse.java
import java.util.Scanner;

public class IfElse{
    public static void main(String[] args){
        try (Scanner scoreScanner = new Scanner(System.in)) {
            boolean continueProgram = true;

            while(continueProgram){
                System.out.println("Get the average of three subjects and grade the student");
                System.out.println("input your scores:");

                System.out.println("Java Score: ");
                Double Java_Score = scoreScanner.nextDouble();

                System.out.println("C Score: ");
                Double C_Score = scoreScanner.nextDouble();

                System.out.println("Database Handling Score: ");
                Double Database_Score = scoreScanner.nextDouble();

                Double AvgScore = (Database_Score + Java_Score + C_Score) / 3.0f;
                String formattedAvg = String.format("%.3f", AvgScore);

                if(AvgScore <= 100 && AvgScore >=90){
                    System.out.println("The average of the student is " + formattedAvg + ", so the student's grade is A.");
                } else if(AvgScore <= 89 && AvgScore >=80){
                    System.out.println("The average of the student is " + formattedAvg + ", so the student's grade is B.");
                } else if(AvgScore <= 79 && AvgScore >=75){
                    System.out.println("The average of the student is " + formattedAvg + ", so the student's grade is C.");
                } else if(AvgScore <=74){
                    System.out.println("The average of the student is " + formattedAvg + ", so the student's grade is F.");
                }
               
                System.out.println("\nDo you want to continue : YES / NO ");
                String choice = scoreScanner.next().trim().toUpperCase();

                if(!choice.equals("YES")){
                    System.out.println("Program terminated.");
                    continueProgram = false;
                }
            }
        }
    }
}

File: IfElse.py
while True:
    print("Get the average of three subjects and grade the student")
    print("Input:")

    Java_Score=int(input("Java Score:"))
    C_Score=int(input("C Score:"))
    Database_Score=int(input("Database Handling Score:"))

    AvgScore = float(Java_Score + C_Score + Database_Score) // 3

    if AvgScore <= 100 and AvgScore >=90:
        print(f"The average of the student is {AvgScore}, so the student's grade is A.")
    elif AvgScore <=89 and AvgScore >=80:
        print(f"The average of the student is {AvgScore}, so the student's grade is B.")
    elif AvgScore <=79 and AvgScore >=75:
        print(f"The average of the student is {AvgScore}, so the student's grade is C.")
    elif AvgScore <=74:
        print(f"The average of the student is {AvgScore}, so the student's grade is F.")

    choice = input("\nDo you want to compute another student's average? (YES/NO): ").strip().upper()

    if choice != "YES":
            print("Program terminated.")
            break